src/serlo-editor/plugins/audio/toolbar.tsx

```diff
@@ -22,6 +28,25 @@ export const AudioToolbar = ({
 }) => {
   const audioStrings = useEditorStrings().plugins.audio
 
+  useHotkeys(
+    [Key.Escape],
+    (event) => {
+      event.preventDefault()
+      setShowSettingsModal(false)
+    },
+    {
+      enableOnContentEditable: true,
+      enableOnFormTags: true,
+      scopes: ['global'],
+    }
+  )
```

Comment:
why is this needed?

Reply:
hmm, works fine for me with e.g. video plugin.
That would be a bug of our Modal Component.